experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/EchoBot.runtimeconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "runtimeOptions": {
```

Comment:
I don't know that this file is needed. I don't think it ever is. None of our other samples have it.

Reply:
removed this and a few other files you commented on. they all came from the template